src/main/java/org/ecommerce/Main.java

```diff
@@ -1,22 +1,31 @@
 package org.ecommerce;
 
-import org.ecommerce.controllers.CustomerController;
 import org.ecommerce.controllers.UserController;
-import org.ecommerce.repositories.CustomerRepository;
-import org.ecommerce.services.CustomerService;
-import org.ecommerce.services.PasswordServiceImpl;
+import org.ecommerce.repositories.UserRepository;
+import org.ecommerce.services.EmailService;
+import org.ecommerce.services.PasswordService;
+import org.ecommerce.services.impl.UserServiceImpl;
+import org.ecommerce.services.impl.EmailServiceImpl;
+import org.ecommerce.services.impl.PasswordServiceImpl;
 
 import java.util.HashMap;
 import java.util.Map;
 
 public class Main {
-    private static final Map<String, Object> REQUEST = new HashMap<>();
-    private static final CustomerRepository customerRepository = new CustomerRepository();
-    private static final UserController customerController = new CustomerController(new CustomerService(customerRepository), new PasswordServiceImpl(customerRepository));
+    private static final Map<String, String> request = new HashMap<>();
+    private static final UserRepository customerRepository = new UserRepository();
+    private static final PasswordService passwordService = new PasswordServiceImpl(customerRepository);
+    private static final EmailService emailService = new EmailServiceImpl();
+    private static final UserServiceImpl customerService = new UserServiceImpl(customerRepository, passwordService, emailService);
+
+    private static final UserController customerController = new UserController(customerService, passwordService);
```

Comment:
Move this logic to test class

Reply:
I just cleared the main method https://github.com/algutierrez-griddynamics/e-commerce/pull/11/commits/a1ea6b262c307306d708b8e3ca63722513c6ddd3